src/nvim/main.c

```diff
@@ -112,6 +113,7 @@ typedef struct {
   int diff_mode;                        // start with 'diff' set
 
   char *listen_addr;                    // --listen {address}
+  int cs_remote;                        // --remote {file1} {file2}
```

Comment:
`mparm_T` implies "command line scan", so the `cs_` prefix isn't necessary.

Reply:
I kept the cs for `clientserver`. thought should give it a prefix because was not sure about the name, i'll change it to `remote` in the next commit. :smile: